hs_core/models.py

```diff
@@ -668,6 +668,31 @@ def remove(cls, element_id):
         raise ValidationError("Description element of a resource can't be deleted.")
 
 
+class Citation(AbstractMetaDataElement):
+    """Define Citation metadata element model."""
+
+    term = 'Citation'
+    value = models.CharField(max_length=750)
```

Comment:
max length of 750 looks a bit arbitrary and may not be enough for some long citation text. Suggest to use text field.

Reply:
Do I just declare a TextField and leave out the max_length specification?

Reply:
Guessing I'll have to re-migrate and that will take care of things, in-place...

Reply:
Yeah, for TextField, you don't have to specify max_length, but you do have to re-migrate.

Reply:
Using TextField